ci/cron/daily-compat.yml

```diff
@@ -116,3 +116,64 @@ jobs:
       - template: ../daily_tell_slack.yml
         parameters:
           success-message: '$(cat $(Build.StagingDirectory)/perf-results.json | jq . | jq -sR ''"perf for ''"$COMMIT_LINK"'':```\(.)```"'')'
+
+  - job: perf_http_json
+    timeoutInMinutes: 120
+    pool:
+      name: "linux-pool"
+      demands: assignment -equals default
+    steps:
+      - checkout: self
+      - bash: ci/dev-env-install.sh
+        displayName: 'Build/Install the Developer Environment'
+      - bash: ci/configure-bazel.sh
+        displayName: 'Configure Bazel for root workspace'
+        env:
+          IS_FORK: $(System.PullRequest.IsFork)
+          # to upload to the bazel cache
+          GOOGLE_APPLICATION_CREDENTIALS_CONTENT: $(GOOGLE_APPLICATION_CREDENTIALS_CONTENT)
+      - template: ../bash-lib.yml
+        parameters:
+          var_name: bash_lib
+      - bash: |
+          set -euo pipefail
+          eval "$(dev-env/bin/dade assist)"
+          source $(bash_lib)
+
+          SCENARIOS="\
+           com.daml.http.perf.scenario.CreateCommand \
+           com.daml.http.perf.scenario.ExerciseCommand \
+           com.daml.http.perf.scenario.CreateAndExerciseCommand \
+           com.daml.http.perf.scenario.AsyncQueryConstantAcs \
+           com.daml.http.perf.scenario.SyncQueryConstantAcs \
+           com.daml.http.perf.scenario.SyncQueryNewAcs \
+           com.daml.http.perf.scenario.SyncQueryVariableAcs \
+          "
+
+          DAR=/var/tmp/quickstart-model.dar
+
+          bazel build //docs:quickstart-model
+          # sandbox cannot open a dar via bazel symlink, copy it into /var/tmp
```

Comment:
I am pretty sure that it does support this. What I suspect is happening is that you are hardcoding a relative path to the DAR and the working directory is changed by `bazel run` so the relative PATH isn’t found. There are two options here:

1. Add the DAR to the runfiles and locate it via `rlocation`. You can find tons of examples for this in our codebase but here is one https://github.com/digital-asset/daml/blob/09014dc63729e4a126306ec22183ded68917eb40/ledger/sandbox-common/src/test/lib/scala/platform/sandbox/AbstractSandboxFixture.scala#L42. This also has the advantage that you do not need the `bazel build` call first. It will be built automatically by Bazel.
2. Use an absolute path. Then `bazel run` won’t mess with it and things should work out. However, you still have to call `bazel build` first.

I’d prefer going with 1 here.

Reply:
how can I use `rlocation` from shell? The way my main is written it accepts file path in the command line argument. Nothing is hardcoded.

Reply:
Fair, in that case, the absolute path is probably the easier option.